File: 최소직사각형.py
def solution(sizes):
    answer = 0
    max_1=[]
    max_2=[]
    for i in sizes:
        if i[0]>i[1]:
            max_1.append(i[0])
            max_2.append(i[1])
        else:
            max_1.append(i[1])
            max_2.append(i[0])
            
    print(max_1)
    print(max_2)
    
    
    return max(max_1)*max(max_2)

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.kt
import java.util.*
class Solution {
    fun solution(sizes: Array<IntArray>): Int {
        var answer: Int = 0
        var first = PriorityQueue<Int>()
        var second = PriorityQueue<Int>()
        sizes.forEach{
            if(it[0]>it[1]){
                first.add(-it[0])
                second.add(-it[1])
            }else{
                first.add(-it[1])
                second.add(-it[0])
            }
        }
        return first.poll() * second.poll()
    }
}